hello-word/src/App.js

```diff
@@ -1,9 +1,11 @@
-import Button from "./components/Button"
 
```

Comment:
It looks good when you remove his blank line

Reply:
i'll try to remove it thanks